kpatch-build/create-diff-object.c

```diff
@@ -2109,6 +2148,9 @@ void kpatch_create_hooks_objname_rela(struct kpatch_elf *kelf, char *objname)
 	}
 }
 
+#ifdef __powerpc__
+void kpatch_create_mcount_sections(struct kpatch_elf *kelf) { }
```

Comment:
Isn't this needed for powerpc? Otherwise, I think patched functions won't be traceable by ftrace.